README.md

```diff
@@ -75,7 +75,7 @@ We have configurated snapshots of FastHub, which can be downloaded from [AppVeyo
   - _FastHub is actively developed. More features will come!_
 
 ~~> **FastHub** contain Ads, which are disabled by default. You could enable them if you'd like to support the development.~~  
-Adverts will not be available until the further notice.
+Ads will not be available until the further notice!
```

Comment:
you forgot to remove `The` it doesn't make any sense to be there.

it should be exactly like: 
Ads are not available until further notice.
Or a better version would be
Ads currently not available.